src/Microsoft.AspNet.Razor/CodeGenerators/Visitors/CSharpTagHelperAttributeValueVisitor.cs

```diff
@@ -102,6 +102,11 @@ protected override void Visit(LiteralChunk chunk)
             RenderCode(chunk.Text, (Span)chunk.Association);
         }
 
+        protected override void Visit(ParentLiteralChunk chunk)
+        {
+            RenderCode(chunk.GetText(), (Span)chunk.Children[0].Association);
```

Comment:
Could refactor `RenderCode` to just take in the start and then you can do `chunk.Start` (assuming it's equivalent to the association's start as mentioned in my comment [here](https://github.com/aspnet/Razor/pull/613/files?diff=split#r45247324)